Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM quay.io/higginsd/resource-downloader
+FROM docker.io/centos:centos7
```

Comment:
This correct? @derekhiggins

Reply:
looks weird to me too, just because of docker.io
anyway it's Dockerfile, so not the ocp related one

Reply:
I don't see in official centos base image on quay.io so used the docker one 
it's the same one we are using in the base image here(docker isn't referenced but that's where its pulled from)
https://github.com/metal3-io/base-image/blob/master/Dockerfile

Reply:
Should we use the RHEL 8 universal base image instead of CentOS?